CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
-- Added registry for backbone models.
+- Added registry for backbone models and losses.
```

Comment:
Since this is likely to raise a backcompatibility issue, I would add a link in the changelog to a documentation release.

Reply:
what do you mean? we do not have docs...

Reply:
We do not, but this is a major change so we should have docs. Could you write something quick for registry before merge?

Reply:
registry stuff is not all done yet, i will do a doc when all is done. this is main not a release. otherwise you will be looking at a PR with 200 file changes...

Reply:
so for docs, i will leave it to another PR/issue